docs/json_schemas/mimir_worker/v0/provider.json

```diff
@@ -0,0 +1,51 @@
+{
+  "title": "ProviderSchema",
+  "description": "Provider schema for Ingress.",
```

Comment:
Ingress?

Reply:
@PietroPasotti can you please regenerate the json schemas from pydantic?